src/test/java/net/snowflake/client/core/arrow/BigIntToTimestampNTZConverterTest.java

```diff
@@ -62,7 +62,16 @@ public void testHonorClientTZForTimestampNTZEnabled() throws SFException {
     testTimestampNTZ();
   }
 
+  @Test
+  public void testWithNullTimezone() throws SFException {
+    testTimestampNTZ(null);
+  }
+
   public void testTimestampNTZ() throws SFException {
+    testTimestampNTZ(TimeZone.getDefault());
+  }
+
+  private void testTimestampNTZ(TimeZone timezone) throws SFException {
```

Comment:
Maybe @Test and `public void` instead of `private void`?

Reply:
This is a helper function for above tests- added a comment in there to clarify that

Reply:
Oh yeah, that looks good!